### PR TITLE
CIGI-180, 181: Populate image and blockquote paragraph types with data

### PR DESCRIPTION
**Pull request by johnbxu**
- Replaced placeholder text with template variables 
- Created basic publication page template for testing paragraphs

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -44,15 +44,17 @@ class BasicPageAbstract(models.Model):
...
-        ('image', ImageChooserBlock()),
-        ('block_quote', blocks.StructBlock([
-            ('quote', blocks.RichTextBlock(required=True)),
-            ('quote_author', blocks.CharBlock(required=False)),
-            ('author_title', blocks.CharBlock(required=False)),
-            ('image', ImageChooserBlock(required=False)),
-            ('link_url', blocks.URLBlock(required=False)),
-            ('link_text', blocks.CharBlock(required=False)),
-        ])),
+        ('image', ImageChooserBlock(template='streams/paragraph_type_image.html')),
```
Comment (gabrieltanchen) [inline core/models.py:47]: Add these to the streams app, similar to how I implemented `ParagraphBlock`, based of [this tutorial](https://learnwagtail.com/tutorials/how-add-basic-streamfield-your-wagtail-cms-page/).

**Review by gabrieltanchen — CHANGES_REQUESTED** [review]

**Comment by johnbxu** [comment]
Created blocks in stream app.

**Comment by gabrieltanchen** [comment]
@johnbxu Sorry for taking so long to get back to this, can you resolve the merge conflicts?

File: templates/streams/image_block.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="paragraph-type-image">
```
Comment (gabrieltanchen) [inline templates/streams/image_block.html:3]: This needs a container.

File: templates/streams/block_quote_block.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="paragraph-type-bloquote">
```
Comment (gabrieltanchen) [inline templates/streams/block_quote_block.html:3]: This needs a container.

File: templates/streams/block_quote_block.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  {% image self.image fill-300x300 %}
+
+  <div class="paragraph-type-blockquote-content">
+    <blockquote>
+      {{ self.quote }}
+    </blockquote>
+
+    {% if self.author %}
+    <div class="paragraph-type-blockquote-author">
```
Comment (gabrieltanchen) [inline templates/streams/block_quote_block.html:12]: Indent within a condition.

File: templates/streams/block_quote_block.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      {{ self.author }}
+    </div>
+    {% endif %}
+
+    {% if self.author_title %}
+    <div class="paragraph-type-blockquote-author-title">
```
Comment (gabrieltanchen) [inline templates/streams/block_quote_block.html:18]: Indent within a condition.

File: templates/streams/block_quote_block.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+      {{ self.author_title }}
+    </div>
+    {% endif %}
+
+    {% if self.link_url %}
+    <a href="{{ self.link_url }}" class="paragraph-type-blockquote-link">
```
Comment (gabrieltanchen) [inline templates/streams/block_quote_block.html:24]: Indent within a condition.

**Review by gabrieltanchen — CHANGES_REQUESTED** [review]
I merged the latest changes in, and dealt with merging the migrations.

I renamed the classes in the `streams` app since "Paragraph" is very Drupal-focused while here we're creating StreamField blocks.

I also created a basic template for article pages so that we can test them.

A couple things I'll leave for you to work on:
1. The block templates need container sizes. We're not setting a container around the whole body since that will make it easier for us to create flows of blocks with varying containers, and even full-bleed containers for images like in the [smart city longform](https://www.cigionline.org/articles/searching-smart-citys-democratic-future).
2. When viewing a page with an image block, Django is returning an error that the `image` template tag is expecting an image but it's being passed a struct. That's because the `ImageBlock` class is a struct, so we need to pass the image within to the template tag instead.

**Comment by gabrieltanchen** [comment]
resolves CIGIHub/cigi-tickets#180
resolves CIGIHub/cigi-tickets#181

**Comment by johnbxu** [comment]
Styled block quote to match live site, and added containers.

**Review by gabrieltanchen — APPROVED** [review]
Great job on this @johnbxu

**Pull request merged**